extension/src/popup/components/manageAssets/TrustlineError/index.tsx

```diff
@@ -78,13 +91,32 @@ const RenderedError = ({
           <div className="TrustlineError__title">
             {t("This asset has a balance")}
           </div>
-          <div className="TrustlineError__body">
+          <div
+            className="TrustlineError__body"
+            data-testid="TrustlineError__body"
+          >
             {t("You still have a balance of")} {assetBalance}.{" "}
             {t("You must have a balance of")} 0{" "}
             {t("in order to remove an asset.")}
           </div>
         </>
       );
+    case TRUSTLINE_ERROR_STATES.ASSET_HAS_BUYING_LIABILITIES:
+      return (
+        <>
+          <div className="TrustlineError__title">
+            {t("This asset has buying liabilities")}
+          </div>
+          <div
+            className="TrustlineError__body"
+            data-testid="TrustlineError__body"
+          >
+            {t("You still have a buying liability of")} {buyingLiabilities}.{" "}
```

Comment:
Should the buying liability message also include the liability currency, like instead of `You still have a buying liability of 1`, should it say `You still have a buying liability of 1 BLND`? 

I'm not sure if that's obvious from the context of that page already.

Reply:
It _should_ be obvious, I think. You have to specifically click `remove asset` next to the asset to get to this screen. 

I'm mostly jsut matching the pattern set by the `this asset has a balance` error above